frontend/src/lib/doc-util.tsx

```diff
@@ -6,10 +6,13 @@ import { TextContent } from '@patternfly/react-core'
 import { useTranslation } from './acm-i18next'
 
 export const DOC_VERSION = '2.13'
```

Comment:
Should the version be 2.12?

Reply:
This PR is for our next release (we've branched already).

Reply:
oh ok then